src/main/java/org/opentripplanner/middleware/models/User.java

```diff
@@ -11,16 +13,54 @@
  */
 public class User extends Model {
     private static final long serialVersionUID = 1L;
-    /** Email address for contact. This must be unique in the collection. */
-    public String email;
     // TODO: Add personal info (name, phone, etc.)
     /**
      * Auth0 user ID, which we initialize to a random value, but when we link this user up with Auth0 we update this
      * value, so the stored user
      */
     public String auth0UserId = UUID.randomUUID().toString();
+
+    /**
+     * Email address for contact. This must be unique in the collection.
+     */
+    public String email;
+
+    /** Whether the user has consented to terms of use. */
+    public boolean hasConsentedToTerms;
+
+    /**
+     * Whether the email address has been verified.
+     * Some identity services, such as Auth0, manage the email verification process and status,
+     * so this field is only for reference.
+     */
+    public boolean isEmailVerified;
+
+    /** Whether the phone number has been verified. */
+    public boolean isPhoneNumberVerified;
+
+    /**
+     * Notification preference for this user
+     * ("none", "email", "sms").
+     * TODO: use enum (https://jira.mongodb.org/browse/JAVA-268)
+     */
+    public String notificationChannel;
+
+    /** Phone number for SMS notifications. */
+    public String phoneNumber;
```

Comment:
Does the User class now contain all required fields? If so, perhaps remove the TODO (on line 16).